src/libs/core/src/internal_functions.cpp

```diff
@@ -709,7 +709,7 @@ DATA *COMPILER::BC_CallIntFunction(uint32_t func_code, DATA *&pVResult, uint32_t
         pV2->SetType(VAR_AREFERENCE);
         pV2->SetAReference(core_internal.Entity_GetAttributePointer(ent));
 
-        if (core.GetEntityPointer(ent))
+        if (core.GetEntityPointerSafe(ent))
```

Comment:
Maybe replace these checks that don't do anything with the return value with an `IsEntityValid` check instead?

Reply:
It's not exposed but it makes sense...